addon/components/frost-icon.js

```diff
@@ -32,7 +32,6 @@ export default Component.extend({
   getDefaultProps () {
     return {
       // options
-      icon: '',
```

Comment:
We should never provide a default for a required property, especially when the default won't render a valid icon to begin with.